package.json

```diff
@@ -11,7 +11,9 @@
     "build:live": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/app.ts",
     "prettier:base": "prettier --parser typescript --single-quote --no-semi",
     "prettier:write": "npm run prettier:base -- --write \"src/**/*.{ts,tsx}\"",
-    "coverage": "nyc npm run test"
+    "coverage": "nyc npm run test",
+    "eslint:fix": "./node_modules/.bin/eslint src/ --ext .ts --fix",
+    "check-code": "npm run prettier:write; npm run eslint:fix"
```

Comment:
`autofix`

Reply:
could we also add a `lint` command and run it in the CI? This way, unlinted code will just get rejected automatically.